portals/knowledge-management/src/Pages/ViewServiceRequest.vue

```diff
@@ -0,0 +1,312 @@
+            <div class="max-w-screen-xl flex flex-col gap-y-6 mx-auto py-8">
+                <div v-if="loadingResults">
+                    <AppLoading />
+                </div>
+                <div v-else>
+                    <main class="flex flex-col gap-8">
+                        <div v-if="searchQuery" class="flex flex-col gap-6">
+                            <SearchResults
+                                :searchQuery="searchQuery"
+                                :searchResults="searchResults"
+                                :loadingResults="loadingeSearchResults"
+                            >
+                            </SearchResults>
+                        </div>
+                        <div v-else class="flex flex-col gap-6">
+                            <Breadcrumbs :currentCrumb="serviceRequest.service_request_number"></Breadcrumbs>
+                            <div class="flex flex-col gap-6">
+                                <h2 class="text-xl font-bold text-primary-950">Service Request Details</h2>
+
+                                <div>
+                                    <div
+                                        class="flex flex-col divide-y ring-1 ring-black/5 shadow-sm px-3 pt-3 pb-1 rounded bg-white mb-4"
+                                    >
+                                        <!-- Service Request Details -->
+                                        <div class="mb-4">
+                                            <h4 class="text-base font-semibold">Title:</h4>
+                                            <p class="text-gray-700">
+                                                {{ serviceRequest.title }}
+                                            </p>
+                                        </div>
+
+                                        <div class="mb-4">
+                                            <h2 class="text-base font-semibold">Description:</h2>
+                                            <p class="text-gray-700">
+                                                {{ serviceRequest.description }}
+                                            </p>
+                                        </div>
+
+                                        <div class="mb-4">
+                                            <h2 class="text-base font-semibold">Service Request Number:</h2>
+                                            <p class="text-gray-700">
+                                                {{ serviceRequest.service_request_number }}
+                                            </p>
+                                        </div>
+
+                                        <div class="mb-4">
+                                            <h2 class="text-base font-semibold">Status:</h2>
+                                            <p class="text-gray-700">
+                                                {{ serviceRequest.status_name }}
+                                            </p>
```

Comment:
It would be good to show this as the tag style with the correct color like on the list view

Reply:
Done